app/api/gradeSimulation/create/func.js

```diff
@@ -0,0 +1,32 @@
+const app = require('@/app')
+const errors = require('@/errors')
+
+module.exports = async function (context) {
+  const GradeSimulation = app.model.gradeSimulations;
+  const HistoryGraduation = app.model.historiesGraduations;
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555384)
```suggestion
  const HistoryGraduation = app.model.historiesGraduations
```